src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -276,7 +295,28 @@ export default {
 		isFocused() {
 			return this.isSearching
 		},
-
+		scrollTopUnread() {
+			this.preventFindingUnread = true
+			this.$refs.scroller.scrollTo({
+				top: this.firstUnreadPos - 100,
+				behavior: 'smooth',
+			})
+			setTimeout(() => {
+				this.handleUnreadMention()
+				this.preventFindingUnread = false
+			}, 500)
+		},
+		scrollBottomUnread() {
+			this.preventFindingUnread = true
+			this.$refs.scroller.scrollTo({
+				top: this.lastUnreadPos - this.$refs.scroller.clientHeight + 150,
```

Comment:
in the mobile apps we always scroll to the first unread mention outside of the viewport, so would be nice to do that here aswell. After clicking it then still shows the button if there are more below the viewport, so that is fine.